build.gradle

```diff
@@ -283,18 +282,11 @@ task pack(type: Tar) {
         from "${dirPack}/rt"
         include '**'
     }
-    into('/aion/web3') {
+    into('/aion/console') {
```

Comment:
Can we find another name for this folder? `console` has a common prefix with `config` leading to a lot of frustrations when trying to auto-complete with tab in the terminal. Perhaps `web-console`?